bip-9999.mediawiki

```diff
@@ -0,0 +1,149 @@
+<pre>
+BIP: 9999
+Title: Redefinition of the Bitcoin Unit to the Base Denomination
+Author: John Carvalho <bitcoinerrorlog@gmail.com>
+Status: Draft
+Type: Informational
+Created: 2024-12-10
+License: CC0-1.0
+</pre>
```

Comment:
Regarding the number assignment, one of the tests will always fail until a number has been assigned and it’s updated in both the table and the BIP. Don’t worry about passing the tests right now. We usually have the preamble indented, and I added the Comments-URI that is required by BIP 2. The Created date is the date a number was assigned, so we will figure that out later.

```suggestion
<pre>
  BIP: ?
  Title: Redefinition of the Bitcoin Unit to the Base Denomination
  Author: John Carvalho <bitcoinerrorlog@gmail.com>
  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-?
  Status: Draft
  Type: Informational
  Created: ?
  License: CC0-1.0
</pre>
```